include/Bishop.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
```

Comment:
Use include guards instead of #pragma once: While #pragma once is widely supported by modern compilers, using include guards (#ifndef, #define, #endif) ensures compatibility with older compilers as well. You can replace #pragma once with include guards in your header files. Example: #ifndef BISHOP_H
#define BISHOP_H

Reply:
Done.